common/reflect_resource_plugin_framework.go

```diff
@@ -62,6 +64,10 @@ func tfSdkToGoSdkSingleField(srcField reflect.Value, destField reflect.Value, sr
 	if srcFieldValue == nil {
 		return nil
 	} else if srcField.Kind() == reflect.Ptr {
+		if srcField.IsNil() {
+			// Skip nils
```

Comment:
Should we have destination field as nil as well or is skipping leads to the same result?

Reply:
Skipping leads to the same result